main.go

```diff
@@ -155,6 +155,13 @@ func loadConfigs() (config.Group, error) {
 		}
 	}
 
+	// Command line Arg
+	if flag.NArg() > 0 {
+		confString := strings.Join(flag.Args(), "\n")
+		confBody := ":" + config.Port + "\n" + strings.Replace(confString, ",", "\n", -1)
```

Comment:
Why won't spaces only suffice? Isn't it what this line handles? `confString := strings.Join(flag.Args(), "\n")`

`$ caddy foo bar baz` should yield three separate strings. `$caddy foo "bar baz"` should yield two separate strings.